examples/rest2/tickers.js

```diff
@@ -2,23 +2,35 @@
 
 process.env.DEBUG = 'bfx:examples:*'
 
-const debug = require('debug')('bfx:examples:rest2_trades')
+const debug = require('debug')('bfx:examples:rest2_tickers')
+const Table = require('cli-table2')
 const bfx = require('../bfx')
 const rest = bfx.rest(2, { transform: true })
 
-debug('fetching data...')
+const table = new Table({
+  colWidths: [10, 14, 14, 14, 14, 14, 14, 18, 18],
+  head: [
+    'Symbol', 'Last', 'High', 'Low', 'Daily %', 'Bid', 'Ask', 'Bid Size',
+    'Ask Size'
+  ]
+})
+
+debug('fetching symbol list...')
 
-rest.tickers(['tBTCUSD', 'tETHUSD'], (err, data) => {
-  if (err) {
-    return debug('error: %j', err)
+rest.symbols().then(symbols => {
+  debug('available symbols are: %s', symbols.join(', '))
+  debug('fetching tickers...')
+
+  return rest.tickers([symbols.map(s => `t${s.toUpperCase()}`)])
+}).then(tickers => {
+  let t
+  for (let i = 0; i < tickers.length; i += 1) {
+    t = tickers[i]
+    table.push([
+      t.symbol, t.lastPrice, t.high, t.low, t.dailyChange, t.bid, t.ask,
+      t.bidSize, t.askSize
+    ])
```

Comment:
order of elements is wrong (see symbol / ask)

![screen shot 2018-04-27 at 12 07 31](https://user-images.githubusercontent.com/298166/39357604-adc997fe-4a13-11e8-87c2-32e2049b7083.png)

Reply:
#298 as well